sphinx_gallery/gen_rst.py

```diff
@@ -13,18 +13,19 @@
 
 """
 from __future__ import division, print_function, absolute_import
+from . import glr_path_static
+from .backreferences import write_backreferences, _thumbnail_div
+from textwrap import dedent
 from time import time
 import ast
+import hashlib
 import os
 import re
 import shutil
-import traceback
-import sys
 import subprocess
+import sys
+import traceback
 import warnings
-from textwrap import dedent
-from . import glr_path_static
-from .backreferences import write_backreferences, _thumbnail_div
```

Comment:
why not keeping the local imports at the end?

it's common to start with std lib then external deps and then local files